### PR TITLE
add translation fot taxonomies and themes

### PR DESCRIPTION
**Pull request by cutterkom**

### CODE REVIEW

File: theme.md [hunk 1]
```diff
@@ -1,20 +1,97 @@
...
+
+- Specific templates for the Repository, Collection and Term item lists but no header images;
+- Specific templates for the item page, including attachment carousel;
+- Standard integration with the Federal Government's link bar and accessibility;
+- Support for most gutenberg blocks;
+- Two menu locations: the main bar and the hidden panel next to it;
+- News carousel on the home page;
+- Customization of sessions on the home page via plugins;
+
+![Algumas telas do Tema IDG-WP](/_assets/gifs/themes-idg-wp.gif)
```
Comment (mateuswetah) [inline theme.md:57]: Some screens from IDG-WP theme

File: theme.md [hunk 2]
```diff
@@ -1,20 +1,97 @@
...
+
+- Specific templates for the Repository, Collection and Term item lists including header and stylized images with some borders and shadows;
+- Item page specific templates, with metadata listed before the Document and attachments;
+- Listing of posts in blocks, in a two-column list;
+- Stylized buttons with wavy effect;
+- Customizable color palette
+- Standard integration with the Federal Government's link holder and accessibility;
+- Integration with the UnB menu bar (if you don't want to use it, just pull from the [branch 'not-unb'](https://github.com/tainacan/tainacan-unb-covid-19/tree/not-unb ":ignore") of GitHub);
+
+![Algumas telas do Tema Veganos](/_assets/gifs/themes-veganos.gif)
```
Comment (mateuswetah) [inline theme.md:71]: Some screens from the Veganos theme

File: theme.md [hunk 3]
```diff
@@ -1,20 +1,97 @@
...
+
+### Rife Free (Tema Filho)
```
Comment (mateuswetah) [inline theme.md:73]: ### Rife Free (Child Theme)

File: theme.md [hunk 4]
```diff
@@ -1,20 +1,97 @@
...
+- Specific templates for the Repository, Collection and Term item lists including header images;
+- Specific templates for the item page, including next and previous buttons, which show their thumbnails;
+- Scroll button to the top;
+- Diverse and customizable color palette;
+- Animations and transition effects;
+- List of posts in blocks, optionally in a list of three columns;
+
+![Algumas telas do Tema Rife Free](/_assets/gifs/themes-rife-free.gif)
+
+### Twenty Twenty (Tema Filho)
```
Comment (mateuswetah) [inline theme.md:86]: ### Twenty Twenty (Child theme)

File: theme.md [hunk 5]
```diff
@@ -1,20 +1,97 @@
...
+
+Every year WordPress releases an "official theme", which is activated by default in its installations and usually has a popular design, with good application of gutenberg blocks in addition to accessibility and responsiveness tests. To validate our tests with the plugin, we created a child theme of [TwentyTweny](https://wordpress.org/themes/twentytwenty/ ":ignore"), available [at this link](https://github.com/tainacan/TwentyTwentyChild ":ignore").
+
+![Algumas telas do Tema Twenty Twenty](/_assets/gifs/themes-twentytwenty.gif)
+
+---
+
+## Gerir Temas no WordPress
```
Comment (mateuswetah) [inline theme.md:94]: ## Manage Themes on WordPress

File: theme.md [hunk 6]
```diff
@@ -1,20 +1,97 @@
...
 
 ### Vídeo Tutorial: Instalação de Temas
```
Comment (mateuswetah) [inline theme.md:96]: ### Video Tutorial: Installing Themes (Portuguese only)

File: taxonomies.md [hunk 7]
```diff
@@ -8,99 +6,100 @@
...
+*Taxonomies* can be created and used for item classification. Examples of common taxonomies are *Gender*, *Countries*, etc.
 
-*Taxonomias* podem ser criadas e usadas para a classificação de itens. Exemplos de taxonomias comuns são a de *Gênero*, *Países*, etc.
+> In *WordPress* language, they are custom taxonomies. See the [WordPress documentation](https://wordpress.org/support/article/taxonomies/) to learn more.
 
-> Na linguagem do *WordPress*, elas são taxonomias personalizadas. Consulte a [documentação do WordPress](https://codex.wordpress.org/pt-br:Taxonomias) para saber mais.
+Each taxonomy has a set of **terms**, for example, gender taxonomy can have terms like drama and comedy.
 
-Cada taxonomia tem um conjunto de **termos**, por exemplo, a taxonomia gênero pode ter termos como drama e comédia.
+The terms may have hierarchy, meaning that when you search for items that have terms with *children* (for example, *Samba*), the results will include items that have any of the terms children. (For example, *Samba de Gafieira* and *Samba Maxixe*).
```
Comment (mateuswetah) [inline taxonomies.md:17]: I hope the example here is clear for _non-brazilians_, maybe using _Rock_, _Progressive Rock_ and _Metal_ would be more universal :rofl:

Comment (cutterkom) [reply taxonomies.md:17]: haha, thought about that, too :)

**Comment by mateuswetah** [comment]
Uhmmm small side note here @cutterkom , I let that pass too: as the images where referenced from the `pt-br` folder we must reference them with the full path (`pt-br/_assets` instead of `_assets`), at least until we don't have some image specifc for english. I'll fix it now. Thanks!

**Comment by cutterkom** [comment]
good to know!

**Pull request merged**